### PR TITLE
Feat/2206 claim streaming payments

### PR DESCRIPTION
**Pull request by davecreaser**
CDapp counterpart: https://github.com/JoinColony/colonyCDapp/pull/2323

* Create a handler for streaming payment claimed.
* Update the DB with the latest claim when a streaming payment is claimed.

### CODE REVIEW

File: src/handlers/expenditures/streamingPaymentClaimed.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  if (!colonyAddress) {
+    return;
+  }
+
+  const streamingPaymentsClient = await getStreamingPaymentsClient(
+    colonyAddress,
+  );
+  if (!streamingPaymentsClient) {
+    return;
+  }
```
Comment (jakubcolony) [inline src/handlers/expenditures/streamingPaymentClaimed.ts:31]: Don't think we need the client at all

**Review by jakubcolony — CHANGES_REQUESTED** [review]
All good apart from one small comment 👍

**Review by jakubcolony — APPROVED** [review]
LGTM, smashing work!

**Review by iamsamgibbs — APPROVED** [review]
Looks good to me! Reviewed with CDapp#2323

**Pull request merged**